package.json

```
@@ -7,14 +7,14 @@
     "stylelint-order": "^6.0.2"
   },
   "engines": {
-    "node": ">=12"
+    "node": ">=16"
   },
   "devDependencies": {
     "standard": "^17.0.0",
     "stylelint": "^15.1.0"
   },
   "peerDependencies": {
-    "stylelint": ">=11"
+    "stylelint": "^14.0.0 || ^15.0.0"
```

Optional:
```suggestion
    "stylelint": ">=14"
```
To avoid warn if the lib is not too maintainted

---

For these I considered using this, but I looked at what other similar libraries were doing and opted for this solution. This way it also matches semver processes as there may be a breaking change for new versions.

---

> For these I considered using this, but I looked at what other similar libraries were doing and opted for this solution. This way it also matches semver processes as there may be a breaking change for new versions.

I agree